api/v1beta1/kafkacluster_types.go

```diff
@@ -537,6 +557,19 @@ func (iIConfig *IstioIngressConfig) GetReplicas() int32 {
 	return iIConfig.Replicas
 }
 
+// GetClientSSLCertSecretName returns the ClientSSLCertSecretName. It returns empty string if It's not specified
+func (k KafkaClusterSpec) GetClientSSLCertSecretName() string {
```

Comment:
This should be *KafkaClusterSpec do not use two separate receiver type for KafkaClusterSpec